### DISCUSSION TITLE
Unlocking more MailKit features

### DISCUSSION THREAD

**Discussion by AaronSadlerUK** [umbraco/Umbraco-CMS#14961] [Features and ideas]
Currently, only the To and From features of MailKit are enabled in core, regularly I have to create a service simply to unlock the ability to use the Reply-To, Bcc, CC options.

I was thinking of creating a PR to unlock these core MailKit features in the Core Umbraco service to remove the need to create a custom service for basic emails

It would be great to get some feedback as I could get this done for hacktoberfest 😅

**Comment by NikRimington** [comment]
I think it would be great to able to expose these, especially CC and BCC.

**Comment by mzajkowski** [comment]
+1

**Comment by AaronSadlerUK** [comment]
I had a look at the IEmailSender and it looks like these have all been added at some point.

Not sure when they got added though
